hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/utils/TestWriteBase.java

```diff
@@ -102,6 +103,23 @@ public class TestWriteBase {
         "id1,par1,id1,Danny,23,3,par1",
         "id1,par1,id1,Danny,23,4,par1",
         "id1,par1,id1,Danny,23,4,par1"));
+
+    removeHadoopConf();
+  }
+
+  private static void removeHadoopConf() {
+    Map<String, String> env = System.getenv();
+    Class<?> clazz = env.getClass();
+    Field field = null;
+    try {
+      field = clazz.getDeclaredField("m");
+      field.setAccessible(true);
+      Map<String, String> map = (Map<String, String>) field.get(env);
+      map.remove("HADOOP_CONF_DIR");
```

Comment:
Should we fix this ? Can you make your local env correct ?

Reply:
@danny0405 Seems a bug in test module, which should also respect HADOOP_CONF_DIR，or in hudi module should not rely on external env?

Reply:
In flink module, we use a static code block for fetching the hadoop configuration, see `HadoopConfigurations.getHadoopConf` for details.

Reply:
Seems test module should also go through HadoopConfigurations.getHadoopConf？if not，test module seems not consistent with hudi core module

Reply:
We should, we can fix the test code if you found the codes that does not use `HadoopConfigurations.getHadoopConf`.

Reply:
Ok, I will fix the test code module later, seems other issues in hudi-flink module I need to fix.